pylintrc

```diff
@@ -60,7 +60,7 @@ disable=
     missing-docstring,
     protected-access,
     too-few-public-methods,
-
+    consider-using-any-all
```

Comment:
I prefer to have these fixed in the codebase rather than disabled. We only disable checks that cannot be solved easily without a lot of code churn.

Reply:
too many errors was coming in just pylint/checkers directory, that's why I disabled it in rc.